### ISSUE TITLE
Enforce end of block brace on new line

### ISSUE THREAD

**Issue by jamespamplin** [jamespamplin/eslint-config-idiomatic#1]
``` js
// Expect error:
while ( true ) {
  i++; }

// Good:
while ( true ) {
  i++;
}
```

**Comment by jamespamplin** [comment]
Raised eslint/eslint#3528

**Comment by jamespamplin** [comment]
Fixed in [ESLint v1.3.0](https://github.com/eslint/eslint/releases/tag/v1.3.0)
